test/cri-containerd/clone_test.go

```diff
@@ -184,7 +186,7 @@ func verifyTemplateContainerState(ctx context.Context, t *testing.T, client runt
 		if templateContainerState == runtime.ContainerState_CONTAINER_EXITED {
 			return
 		}
-		time.Sleep(500 * time.Millisecond)
+		time.Sleep(1 * time.Second)
```

Comment:
Were we seeing issues with this timeout being too short?

Reply:
Yeah, the test was intermittently failing because of this. So I increased this.